web/pgadmin/tools/sqleditor/templates/sqleditor/sql/default/insert.sql

```diff
@@ -4,7 +4,7 @@ INSERT INTO {{ conn|qtIdent(nsp_name, object_name) | replace("%", "%%") }} (
 {% if not loop.first %}, {% endif %}{{ conn|qtIdent(col) | replace("%", "%%") }}{% endfor %}
 ) VALUES (
 {% for col in data_to_be_saved %}
-{% if not loop.first %}, {% endif %}%({{ pgadmin_alias[col] }})s{% if type_cast_required[col] %}::{{ data_type[col] }}{% endif %}{% endfor %}
+{% if not loop.first %}, {% endif %}{% if data_to_be_saved[col] == 'set_default'%} DEFAULT {% else %}%({{ pgadmin_alias[col] }})s{% if type_cast_required[col] %}::{{ data_type[col] }}{% endif %}{% endif %}{% endfor %}
```

Comment:
Why are you changing the backend code? This was handled in front end and can be fixed in front end itself.

Reply:
I had discussion with akshay and nikhil regarding this. We need at least one value to be provided from front end for insert to work so, if a user does not provide any values insert will fail so we need to handle it in backend only.

Reply:
@pravesh-sharma OK understood.
But I think this code might not work in case where:
1. I have a char varrying column with some default value
2. I add a row, and set that column data to `set_default` in front end and save.
3. It will take default value instead of `set_default`.
Can you verify this scenario?